modules/bullet/area_bullet.cpp

```diff
@@ -94,6 +94,9 @@ void AreaBullet::dispatch_callbacks() {
 				otherObj.object->on_exit_area(this);
 				overlappingObjects.remove(i); // Remove after callback
 				break;
+			case OVERLAP_STATE_DIRTY:
+			case OVERLAP_STATE_INSIDE:
+				break;
```

Comment:
```

modules/bullet/area_bullet.cpp:86:10: warning: enumeration value 'OVERLAP_STATE_DIRTY' not handled in switch [-Wswitch]
modules/bullet/area_bullet.cpp:86:10: warning: enumeration value 'OVERLAP_STATE_INSIDE' not handled in switch [-Wswitch]
```
@AndreaCatania

Reply:
I don't like add two more useless check, is it possible hide this warning in another way?

Reply:
@AndreaCatania If this is a performance-wise concern, note that such cases are going to be optimized at compilation time. So such check have no impact on performances.

Reply:
I don't want to disable the `-Wswitch` warning as it does uncover real bugs. So all `switch`s should handle all their enum values, either like done above, or via `default: {}` as done in some other places. Using `default: {}` means that if in the future other enum values are added and they were meant to be handled in the switch, we won't be told about it by the compiler.

The other alternative is to not use a `switch` if you only want to handle some values and not all of them (so use `if`/`else if` instead).